org-code-javabuilder/playground/src/main/java/org/code/playground/Board.java

```diff
@@ -0,0 +1,124 @@
+    /**
+     * Returns the width of the playground screen. This will always be 400.
+     * 
+     * @return the width of the playground in pixels.
+     */
+    public int getWidth();
+
+    /**
+     * Returns the height of the playground screen. This will always be 400.
+     * 
+     * @return the height of the playground in pixels.
+     */
+    public int getHeight();
+
+    /**
+     * Sets the background of the playground to the provided image. The image will
+     * be scaled to fit the full playground screen, which may distory the image if
+     * it is not square.
+     * 
+     * @param filename the name of the file from the asset manager to put in the
+     *                 background
+     * @throws FileNotFoundException if the file cannot be found in the asset
+     *                               manager
+     */
+    public void setBackgroundImage(String filename) throws FileNotFoundException;
+
+    /**
+     * Adds a clickable item from the playground.
+     * 
+     * @param item the item to add. If the item is already in the playground,
+     *              this method does nothing.
+     * @param x the distance, in pixels, from the left side of the playground
+     * @param y the distance, in pixels, from the top of the playground
+     */
+    public void addItem(ClickableItem item, int x, int y);
+
+    /**
+     * Removes the clickable item from the playground.
+     * 
+     * @param item the item to remove. If the item is not in the playground, this
+     *              method does nothing.
+     */
+    public void removeClickableItem(ClickableItem item);
+
+    /**
+     * Adds a non-clickable item to the playground.
+     * 
+     * @param item the item to add. If the item is already in the playground,
+     *              this method does nothing.
+     * @param x the distance, in pixels, from the left side of the playground
+     * @param y the distance, in pixels, from the top of the playground
+     */
+    public void addItem(Item item, int x, int y);
+
+    /**
+     * Removes the non-clickable item from the playground.
+     * 
+     * @param item the item to remove. If the image is not in the playground, this
+     *              method does nothing.
+     */
+    public void removeItem(Item item);
+    
+    /**
+     * Draws text on the playground.
+     *
+     * @param text the text to draw
+     * @param x the distance from the left side of the playground to draw the text.
+     * @param y the distance from the top of the playground to draw the text.
+     * @param color the color to draw the text.
+     * @param font the name of the font to draw the text in.
+     * @param fontStyle the name of the font style to draw the text in.
+     * @param height the height of the text in pixels.
+     * @param rotation the rotation or tilt of the text, in degrees
+     */
+    public drawText(String text, int x, int y, Color color, Font font, FontStyle fontStyle, int height, double rotation);
```

Comment:
How are we planning to avoid confusion relating to the media package? i.e. I know we had decided to not extend Image to make clickableImage, but we're still having them import Color, Font, and FontStyle from the media package to use drawText.

Reply:
I think I misspoke about avoid confusion with the media package. I don't think there will be confusion in using the utilities we've created in `org.code.media` - I think of it like our own version of `java.awt.Color`. The confusion to avoid is overlap in capability with Theater, so including the full `Image` class in Playground could cause that kind of confusion.

Reply:
👍

Reply:
Another text question - would we need a way to remove text, or is text permanent once drawn?